packages/nimble-components/src/table/specs/table-column-specs/table-column-mapping-anchor.md

```diff
@@ -0,0 +1,197 @@
+# Mapping Anchor Table Column
```

Comment:
Once this is closer to ready for approval, we should add Stefan as a required reviewer.